### DISCUSSION TITLE
Notification link points to 404 when on gist.github.com

### DISCUSSION THREAD

**Discussion by unrealwill** [community/community#20195] [Projects and Issues] [answered]
When I open a page that is on the subdomain https://gist.github.com the notification bell button link near my account redirect to https://gist.github.com/notifications which 404 instead of redirecting towards https://github.com/notifications

Thanks

**Comment by MrKrishnaAgarwal** [comment]
It will be resolved, maybe a glitch.

**Comment by radar-blip** [comment, marked as answer]
Try going to your profile before clicking on notifications. Works for me. Probably just a small glitch.  
Or you can report it as a bug here:  https://support.github.com/contact/bug-report

**Reply by unrealwill** [reply]
@linuxlawson Thanks, I thought the discussion page was read by people from github but given that this trivial bug is still there it mustn't be the case. I have just filed a bug report on the link you provided. Hopefully it will get fixed.

**Reply by radar-blip** [reply]
Its been reported a few times that I know of. Not a big deal but certainly fixable.
Have a great day!

**Reply by radar-blip** [reply]
Update: Seems to be fixed now.
